README.md

```diff
@@ -1,15 +1,42 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  - constant factors and hidden constants typically in asymtotic analysis constant factors are ignored.
+    In real life situation these factors can have a decent impact on the actual time comlexity.
+  - The input data given in the real world can be much different then what it would be assumed to be for the asymtotic analysis.
+    an algorrithm that performs well for the average and worst case senario might might not have good performance for a
+    specific input that was not considered.
```

Comment:
Ok, but then the analysis was wrong.

Reply:
Yes, It seems that would make the analysis incorrect. I think that another reason would be that the type of analysis you are doing like if it is big O notation then this would give you loose bounds which mean that the runtime my be faster than the analysis is for the worst case scenario.